### ISSUE TITLE
Improvement suggestions after accessibility workshop

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#2840]
@BorghildSelle - adding everything here, as agreed

•	Remove "Screenreader label" field from CTA links. 
- [x] 	File icons updates

The pdf icon needs an "accessible name"
Suggestion: in the code it says aria-hidden=true, should have ari-label="pdf"
We also need icons for excel and xbrl

- [x] The download/internal/external icons don’t give the correct information. 

Should have aria-labels internal-link, eksternal-link, download-document
•	Magazin tags: are made as list elements but are without content. Either remove the list-element or make the divs within to list elements. Ul-tag could be changed to div or span, or role="list-item" (easiest solution?). Can’t use list without list elements
•	Recaptcha-logoen: make bigger padding around the logo element (Interactive element does not meet minimum size nor spacing)
•	Change the H3 i promotiles and topic promo link text. Maybe create a new class? Not correct with regards to SEO to have a heading without content following it.
•	Red H2 doesn’t have a good enough contrast. Could it be bold – or just live it?
Something wrong (nothing happening) after Submit search 
•	Update «Our» videocomponent – with regards to issue with scrollable element…
Change tabindex from -1 til 0
•	Sticky menu: can use to much real estate. Recommended to avoid on mobile.
Screenreader issue that when clicking on an anchor link you go down to the right area further down, but focus for the screenreader is still at the top. So it doesn’t work like a «real» anchor link (you continue tabbing from the sticky menu at the top of the page)

**Comment by BorghildSelle** [comment]
Moving this to its own task :)
Html video element should be able to give a source (Error message: No compatible source was found for this media.)
More info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

**Comment by meols** [comment]
@BorghildSelle - adding this to a separate card: 
_Textlink in accordion -
Bring back the hyperlink on text option (also keep link button after the text) – to have links in context of content._

**Comment by millianapia** [comment]
note regarding this:

> • Recaptcha-logoen: make bigger padding around the logo element (Interactive element does not meet minimum size nor spacing)

At the moment, there is no way to fully style the reCAPTCHA elements (including logo), only the wrapper elements around the iframe can be stylized
